pkg/minikube/registry/drvs/docker/docker.go

```diff
@@ -58,10 +58,20 @@ func init() {
 }
 
 func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
+	mounts := make([]oci.Mount, len(cc.VolumeMounts))
```

Comment:
I believe we can add same exact code for podman driver. registry/drvs/podman/podman.go

Reply:
I agree, will do!